templates/ccm_secret.yaml

```diff
@@ -1,3 +1,18 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: metal-cloud-config
+  namespace: kube-system
+stringData:
+  cloud-sa.json: |
+    {
+      "apiKey": "${auth_token}",
+      "projectID": "${project_id}"
```

Comment:
For the HA API loadbalancer, per https://github.com/equinix/cloud-provider-equinix-metal#control-plane-loadbalancer-implementation, I suspect we would need:
```
    "loadbalancer": "kube-vip://"
```

Reply:
So... This is working without that today... Does this actually install and manage Kube-VIP for me? (Which I don't know if I want, since I need Kube-VIP to be alive before the cluster creation can fully complete)

Reply:
Hmm... looks like no. Let's break off the load balancing touchups into the next version, since this version is already working as is.